model/model_training/utils.py

```diff
@@ -296,12 +299,14 @@ def get_model(conf, tokenizer, pad_vocab_size_to_multiple_of=16):
 
     if conf.is_reward_model:
         if "pythia" in conf.model_name:
-            model = GPTNeoXRewardModel.from_pretrained(conf.model_name, cache_dir=conf.cache_dir, torch_dtype=dtype)
```

Comment:
hm. why was the dtype specification here removed .. ?

Reply:
mainly due to unable to rescale fp16, but as you mentioned, we could just assign fp32 for that